visitor-app/src/cb_admin/pages/VisitsData/index.js

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import moment from 'moment';
+import { assocPath, compose } from 'ramda';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
Can we please not use this library

Reply:
Actually on this, going back to your [reasoning here](https://github.com/TwinePlatform/twine-monolith/issues/324#issuecomment-544917966) for avoiding use of this library, I've discovered that the reason we couldn't overwrite the padding/margin was because we were importing the `index.css` file before the `App.js` file. I've change the order now, and have been able to successfully overwrite the default padding using an attribute selector:
```css
[class*="col-"] {
  padding: 0;
}
```
So I think we're good. Also saves me a heap of time since everything is already layed out.

Reply:
🧂